File: proyectoJava/src/civitas/TestP2.java
package civitas;

import java.util.ArrayList;

public class TestP2 {
    public static void main(String[] argv) {
        System.out.println("TestP2:");
        
        ArrayList<String> nombres = new ArrayList<>(4);
        for (int i = 0; i < 4; i++){
            String nombre = "Jugador" + i;
            nombres.add(nombre);
        }
        
        CivitasJuego juego = new CivitasJuego(nombres);
        
        System.out.println(juego.infoJugadorTexto());
    }
}

File: proyectoRuby/lib/test_p2.rb
module Civitas

require_relative 'civitas_juego'
  
class TestP2

  def initialize
    main
  end
  
  def main
    puts "TestP2:\n"
    
    nombres = []
    for i in 0..4 do
      str = "Jugador" + i.to_s
      nombres.push(str)
    end
    
    juego = CivitasJuego.new(nombres)
    puts juego.info_jugador_texto
    
    
    
    
  end
end

TestP2.new
end
